galleries/examples/misc/contour_manual.py

```diff
@@ -43,11 +44,12 @@
 # Multiple filled contour lines can be specified in a single list of polygon
 # vertices along with a list of vertex kinds (code types) as described in the
 # Path class.  This is particularly useful for polygons with holes.
-# Here a code type of 1 is a MOVETO, and 2 is a LINETO.
 
 fig, ax = plt.subplots()
 filled01 = [[[0, 0], [3, 0], [3, 3], [0, 3], [1, 1], [1, 2], [2, 2], [2, 1]]]
-kinds01 = [[1, 2, 2, 2, 1, 2, 2, 2]]
+M = Path.MOVETO
+L = Path.LINETO
+kinds01 = [[M, L, L, L, M, L, L, L]]
```

Comment:
While at it, I would go a bit more verbose:

```suggestion
MOVE = Path.MOVETO
LINE = Path.LINETO
kinds01 = [[MOVE, LINE, LINE, LINE, MOVE, LINE, LINE, LINE]]
```

or even

```
steps = [
    (Path.MOVETO, [0, 0]),
    (Path.LINETO, [3, 0]),
    (Path.LINETO, [3, 3]),
    (Path.LINETO, [0, 3]),
    (Path.MOVETO, [1, 1]),
    (Path.LINETO, [1, 2]),
    (Path.LINETO, [2, 2]),
    (Path.LINETO, [3, 1]),
]

fills, kinds = zip(*steps)
```

But the proposal is also ok if you don't want to do more.

Reply:
I don't think long names really help readability here; the point is really to see that there's 1 M followed by 3 Ls, twice.

Reply:
I think that some way to "visually" connect the code and the points would be nice, but I am not really sure the zip-thing is worth it. So may just keep it as is.